### ISSUE TITLE
SCloss

### ISSUE THREAD

**Issue by Wy-98** [siyueyu/DCFM#2]
Thanks for your work.
But the SC Loss seems not to converge based on the related code. Can the code available achieve comparable  performance to that in paper?

**Comment by siyueyu** [comment]
@Wy-98  You can refer to the log.txt in the pretrianed file or reproduction file.

**Comment by Wy-98** [comment]
We know it converges as in the log.txt. But it seems to fail when we train the model with the available code. So we wonder whether some other techniques are needed to achieve the preformance in your paper.
Thanks for your time.

**Comment by siyueyu** [comment]
@Wy-98 Did you use the same training dataset COCO_seg?

**Comment by Wy-98** [comment]
Yes, we use the COCO_seg to train the model.

**Comment by Wy-98** [comment]
[log.txt](https://github.com/siyueyu/DCFM/files/8771112/log.txt)
This is the log.txt.

**Comment by siyueyu** [comment]
@Wy-98 I will check the code again. You can also try to train the network without scl. Let me know the result if you tried this situation. For me, the result can reach 58 of F-measure for CoCA without scl.

**Comment by siyueyu** [comment]
@Wy-98  what environment do you use?

**Comment by siyueyu** [comment]
@Wy-98 I tried the uploaded code but I haven't dug deep. In my attempt, the loss can converge. Maybe you can try different seeds in the set_seed(). For other information, I will keep informing you if I find anything. For now, due to the covid-19, I can't attend my lab, so it will be difficult and inconvinient for me to conduct more researches. Sorry for the unstable code.

**Comment by Wy-98** [comment]
Sorry for the late reply.
I tried to train the model without scl. The corresponding F-measure core is about 55.5.

environment: torch-1.9.1; torchvision-0.8.2

I will further try different seeds to see whether the results could be better. Thanks for your answer.

**Comment by Wy-98** [comment]
Sorry for the late reply.
I tried to train the model without scl. The corresponding F-measure core is about 55.5.

environment: torch-1.9.1; torchvision-0.8.2

I will further try different seeds to see whether the results could be better. Thanks for your answer.

**Comment by siyueyu** [comment]
@Wy-98 I only tried torch-1.7.1 torch.vision-0.9.1.

**Comment by Wy-98** [comment]
Thanks a lot. And I will try to carry out the experiments with torch-1.7.1 and torch.vision-0.9.1.
